code/modules/discord/tgs_commands.dm

```diff
@@ -1,17 +0,0 @@
-// Notify
-/datum/tgs_chat_command/notify
```

Comment:
Just to double check, removing this and `toggle_notify.dm` is intentional and not just a carryover from Horizon?

Reply:
I believe it's both, due to a restructuring of the way SSDiscord operates. It's not *really* worth reimplimenting right now either.